src/python/pants/engine/target.py

```diff
@@ -1593,9 +1599,6 @@ def rules():
     input: ClassVar[Type[Sources]]
     output: ClassVar[Type[Sources]]
 
-    def debug_hint(self) -> str:
-        return "{self.protocol_target.address.spec}"
-
```

Comment:
Why remove?

Reply:
It was a broken format string, which likely means that no-one has ever actually encountered it rendering in the wild.

Reply:
Hm, or no one reported it. Note that pantsbuild/pants doesn't use codegen, and users are less likely to report things. Imo it's valuable to keep it